hw_3/for_database.py

```diff
@@ -0,0 +1,70 @@
+from sqlalchemy import create_engine, Table, Column, Integer, String, MetaData
+from sqlalchemy.orm import mapper
+
+engine = create_engine('sqlite:///sqlite3_project.db', echo=True)
+
+class Storage:
+
+    def __init__(self, id, login, password, information, message, time_send, ip, id_owner, id_contact):
+        self.id = id
+        self.login = login
+        self.password = password
+        self.information = information
+        self.message = message
+        self.time_send = time_send
+        self.ip = ip
+        self.id_owner = id_owner
+        self.id_contact = id_contact
+
+
+metadata = MetaData()
+clients_table = Table('clients', metadata,
+                      Column('id', Integer, primary_key=True),
+                      Column('login', String),
+                      Column('information', String)
+                      )
+
+
+class Clients:
+    def __init__(self, login, information):
+        self.login = login
+        self.information = information
+
+    def __repr__(self):
+        return f'Storage {self.login}, {self.information}'
```

Comment:
```suggestion
        return f'<Client {self.login}, {self.information}>'
```
У объектов pyton результат `__repr__` примерно похожий, лучше не менять форму.